its/projects/CSharp.SDK.2.1/AspNetCoreMvc/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - AspNetCoreMvc5</title>
```

Comment:
Looks like a copy/paste artifact. All comments in .NET  version applies to 2.1, 3.1 and Latest as well so it will go away.
```suggestion
    <title>@ViewData["Title"] - AspNetCoreMvc</title>
```